packages/extension-polkagate/src/util/workers/sharedWorker.js

```
@@ -9,49 +11,56 @@
-        break;
+    try {
+      switch (functionName) {
+        case 'getAssetOnRelayChain':
+          getAssetOnRelayChain(...params, port).catch(console.error);
+          break;
 
-      case 'getAssetOnMultiAssetChain':
+        case 'getAssetOnMultiAssetChain':
         // eslint-disable-next-line no-case-declarations
-        const assetsToBeFetched = assetsChains[parameters.chainName];
+          const assetsToBeFetched = assetsChains[parameters.chainName];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)